packages/l/libsdl_gfx/xmake.lua

```diff
@@ -25,13 +25,24 @@ package("libsdl_gfx")
         add_extsources("brew::sdl2_gfx")
     end
 
-    add_deps("libsdl")
+    if is_plat("wasm") then
+        add_configs("shared", {description = "Build shared library.", default = false, type = "boolean", readonly = true})
+        add_cxflags("-sUSE_SDL=0")
```

Comment:
since libsdl_gfx (and others) depends on libsdl which adds this flag already, isn't this redundant?